ping_server/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10
```

Comment:
Noticed we had Python 3.12 above in the Circle CI config.yml:
https://github.com/mozilla/klaatu/blob/a3102f0c3a962e8f665bec06f468082379832cd2/.circleci/config.yml#L34-L36

Not sure it matters, but the eyes see what the eyes see.